### PR TITLE
Move ke_c_sw to c_sw

### PR DESCRIPTION
**Pull request by jdahm**

### CODE REVIEW

File: fv3core/stencils/c_sw.py [hunk 1]
```diff
@@ -252,7 +292,38 @@ def compute(delp, pt, u, v, w, uc, vc, ua, va, ut, vt, divgd, omga, dt2):
         origin=geo_origin,
         domain=(grid.nic + 2, grid.njc + 2, grid.npz),
     )
-    ke, vort = ke_c_sw.compute(uc, vc, u, v, ua, va, dt2)
+
+    # ke_c_sw
+    # {
+    # Create storage objects to hold the new vorticity and kinetic energy values
```
Comment (rheacangeo) [inline fv3core/stencils/c_sw.py:298]: Thank you for the comment!

File: fv3core/stencils/c_sw.py [hunk 2]
```diff
@@ -252,7 +292,38 @@ def compute(delp, pt, u, v, w, uc, vc, ua, va, ut, vt, divgd, omga, dt2):
...
+        grid.cos_sg3,
+        grid.sin_sg4,
+        grid.cos_sg4,
+        dt2,
+        origin=(grid.is_ - 1, grid.js - 1, 0),
+        domain=(grid.nic + 2, grid.njc + 2, grid.npz),
+    )
+    # }
+
+    # ke, vort = ke_c_sw.compute(uc, vc, u, v, ua, va, dt2)
```
Comment (rheacangeo) [inline fv3core/stencils/c_sw.py:326]: should we delete this since it is no longer used?

**Review by rheacangeo — APPROVED** [review]
Looks great! way to consolidate

File: fv3core/stencils/c_sw.py (hunk 2 above)
Comment (jdahm) [reply fv3core/stencils/c_sw.py:326]: Yeah, I'll remove this in a follow-up PR, as I'm still working in `c_sw.py` for the next few in this series.

**Pull request merged**